composer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "friendsofphp/number",
+    "description": "Number utility for displaying human-friendly numeric data",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "FriendsOfPhp\\Number\\": "src/"
+        }
+    },
+    "authors": [
+        {
+            "name": "Caen De Silva",
+            "email": "caen@desilva.se"
+        }
+    ],
+    "require": {
+        "php": "^8.1 <8.4",
```

Comment:
**I bumped this to 8.1, is that acceptable?**

Context:

While the code should run fine on 8.0, the libraries we use for testing do not, and I don't think a work around is worth it since 8.0 is no longer supported and 8.1 is the minimum official PHP version.

Reply:
For wider adoption it would be better to support older PHP versions, even 7.4, but it would make everything more complex - CI would have to be configured to run on more PHP versions, dependenciec would have to be downgraded to versions that support older PHP versions, and of course package's code itself wouldn't be able to use newer language features. Hard to tell 🤷‍♂️.

There is always option for downgrading the code using Rector before releasing version, but this is another way of making process more complex 😉.

Reply:
Yup, there are always tradeoffs to consider. Maybe a compromise of leaving it as it is now for the initial release, and if we get people actually requesting a backport we can revisit the matter then?